setup.cfg

```diff
@@ -54,6 +54,12 @@ install_requires =
     pandas
     linkml
     scipy
+    validators
+    networkx
+    pandasql
+    sparqlwrapper
+    jsonasobj~=1.2.1
```

Comment:
is this required? isn't this a transitive dependency? 

I feel this may have been put in as a patch at some point. I think if removed everything will work fine

Reply:
I would say including transitive dependencies in setup.cfg is explicitly bad. I didn't carefully read all requirements list but rather copied this over from the requirements.txt file, so I would be happy to remove ones that aren't explicitly imported in the package itself. However, any dependency that's explicitly imported in a package should explicitly list it as a dependency. I hope the difference makes sense, let me know if not and I will elaborate